src/utils/map-state.js

```diff
@@ -77,27 +63,25 @@ export default class MapState {
     /** Zoom when current zoom operation started */
     startZoom
   } = {}) {
-    assert(Number.isFinite(width), '`width` must be supplied');
-    assert(Number.isFinite(height), '`height` must be supplied');
-    assert(Number.isFinite(longitude), '`longitude` must be supplied');
-    assert(Number.isFinite(latitude), '`latitude` must be supplied');
-    assert(Number.isFinite(zoom), '`zoom` must be supplied');
+    assert(isFinite(width), '`width` must be supplied');
```

Comment:
I think we should make some effort to be consistent in our pick of what function to use to validate parameters.

The problem with the older `isFinite` (rather than `Number.isFinite`) is that it coerces its argument to a number. Meaning that many non-numeric parameters will be accepted.

Browser console:
```
isFinite('')
true
isFinite(null)
true
isFinite('1')
true
isFinite('a')
false
```